frontend/src/_constants.scss

```
@@ -6,6 +6,8 @@ $red-samf-faded: #fac7c8;
 $red-lighter: #fff2f0;
 $red-light: #ffcfca;
 $red: #dc1010;
+$salmon: #fa8072;
+$salmon_light: #fab4ac;
```

Follow naming convention

```suggestion
$salmon-light: #fab4ac;
```

---

fixed